pkg/controllers/dynakube/extension/tls/reconciler_test.go

```diff
@@ -55,13 +56,14 @@ func TestReconcile(t *testing.T) {
 
 		reconciler := NewReconciler(fakeClient, fakeClient, dk)
 
-		reconciler.Reconcile(context.Background())
+		err := reconciler.Reconcile(context.Background())
+		require.NoError(t, err)
 
 		var secret corev1.Secret
-		err := fakeClient.Get(context.Background(), SelfSignedTLSSecretObjectKey, &secret)
+		fakeClient.Get(context.Background(), SelfSignedTLSSecretObjectKey, &secret)
 
 		require.NoError(t, err)
```

Comment:
if you don;t return error on line 61, why we need to check error here? 🤔

Reply:
[Fixed](https://github.com/Dynatrace/dynatrace-operator/pull/3950/commits/ad38cb1a1bdfe0821d57f20acd42c9e18c03f49d), missed it when reverting other changes